watson_developer_cloud/discovery_v1.py

```diff
@@ -402,3 +402,262 @@ def query(self, environment_id, collection_id, query_options):
             url='/v1/environments/{0}/collections/{1}/query'.format(
                 environment_id, collection_id),
             params=query_options, accept_json=True)
+
+
+    def delete_training_data(self, environment_id, collection_id):
+        """
+        Clears all training data for this collection.
+        :param environment_id: the ID of your environment
+        :param collection_id: the ID of your collection
+        :return: Code: 204, All training data removed
+        """
+        format_string = '/v1/environments/{0}/collections/{1}/training_data'
+        url_string = format_string.format(environment_id, collection_id)
+        params = {'version': self.version}
+
+        return self.request(method='DELETE',
+                            url=url_string,
+                            params=params)
```

Comment:
Ideally the service itself would return JSON here, even if just an empty object or a message with the ID of the deleted resource. (For how the service is now, the SDK is fine.)